apps/climatemappedafrica/src/payload/blocks/DataIndicators.js

```diff
@@ -0,0 +1,66 @@
+import image from "../fields/image";
+import richText from "../fields/richText";
+
+const DataIndicators = {
+  slug: "data-indicators",
+  imageURL: "/images/cms/blocks/data-indicators.png",
+  imageAltText: "Used in homepage",
+  labels: {
+    singular: "Data Indicator",
+    plural: "Data Indicators",
+  },
+  fields: [
+    {
+      name: "title",
+      type: "text",
+      label: "Title",
+      required: true,
```

Comment:
1. `Title` here looks the same as one in `PageHero`/`Hero`... lets use the same config + front-end component to render it. I think it's `RichHeader` or something.
2. Fields should have `localized: true` similar to how they're defined in other blocks/pages.